qunetsim/components/network.py

```diff
@@ -523,6 +523,17 @@ def _process_queue(self):
                     continue
 
                 sender, receiver = packet.sender, packet.receiver
+                sending_host = self.get_host(sender)
+
```

Comment:
Look into _route_quantum_info, there we currently have error parameters. These have to be removed from the network. I also would put the channel part into this function, not here.

Reply:
Error parameters have been removed.
Application of channel properties has been moved to transfer_qubits function in the _route_quantum_info method.

I would also like to point out that though _route_quantum_info takes a list of qubits as a parameter, it is never called with a list containing more than 1 qubit. Can you clarify as to why a list is required?

Reply:
@HalcyonicFalconX  This is kind of "future proofing" thing I thought about a while ago. I thought there might come a time when we want to add a feature for "sending a list of qubits" at some point, therefore the network takes a list of qubits instead of 1 qubit.